server/src/db/dal/items.dal.ts

```diff
@@ -34,4 +34,17 @@ export const getItemsByCategoryAndUserId = async (categoryId: mongoose.Types.Obj
 
     const items : Item[] = await ItemModel.find(query).exec();
   return await getAllItemsWithS3Images(items);
-};
\ No newline at end of file
+};
+
+export const deleteItemById = async (itemId: mongoose.Types.ObjectId): Promise<void> => {
+    await ItemModel.deleteOne({_id: itemId}).exec();
+  }
+  
+export const editItemById = async (itemId: mongoose.Types.ObjectId, updatedItem: Item): Promise<Item> => {
+    const item = await ItemModel.findOneAndUpdate({_id: itemId}, updatedItem, {new: true}).exec();
+    if (!item) {
```

Comment:
Because of validations you'll add, you won't need this throw.
You can make this method one line: `return ItemModel.findOneAndUpdate(.....`